### PR TITLE
add integration springCloud test

### PR DESCRIPTION
**Pull request by runqi-zhao**
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-integrated-test/shenyu-integrated-test-k8s-ingress-spring-cloud/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-k8s-ingress-spring-cloud/docker-compose.yml:1]: Why need docker-compose?

File: .github/workflows/integrated-test-k8s-ingress-spring-cloud.yml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+name: it-k8s
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  spring-cloud-ingress-controller:
```
Comment (JooKS-me) [inline .github/workflows/integrated-test-k8s-ingress-spring-cloud.yml:25]: ingress-controller-spring-cloud

File: .github/workflows/integrated-test-k8s-ingress.yml
Comment (JooKS-me) [inline .github/workflows/integrated-test-k8s-ingress.yml:1]: change job name from `ingress-controller` to `ingress-controller-http`

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: .github/workflows/integrated-test-k8s-ingress.yml
Comment (runqi-zhao) [reply .github/workflows/integrated-test-k8s-ingress.yml:1]: ok, do you need to change the original ingress-controller (e.g. shenyu-integrated-test-k8s-ingress) to an ingress-controller-http form?

**Review by JooKS-me — APPROVED** [review]

**Pull request merged**